### PR TITLE
cherrypicked from master tls secrets format

### PR DESCRIPTION
**Pull request by nmnellis**
**Please provide a description of this PR:**

cherry pick https://github.com/istio/istio/pull/39732

### CODE REVIEW

**Comment by ericvn** [comment]
```
# istio.io/istio/security/pkg/pki/ca
security/pkg/pki/ca/ca.go:251:3: undefined: fileBundle
security/pkg/pki/ca/ca.go:258:24: undefined: fileBundle

real	1m10.554s
user	12m11.285s
sys	1m37.007s
make: *** [Makefile.core.mk:271: build] Error 2
```

**Comment by ericvn** [comment]
lint test has many others...

File: security/pkg/pki/ca/ca.go [hunk 1]
```diff
@@ -235,14 +248,14 @@ func NewPluggedCertIstioCAOptions(certChainFile, signingCertFile, signingKeyFile
 	}
 
 	if caOpts.KeyCertBundle, err = util.NewVerifiedKeyCertBundleFromFile(
-		signingCertFile, signingKeyFile, certChainFile, rootCertFile); err != nil {
+		fileBundle.SigningCertFile, fileBundle.SigningKeyFile, fileBundle.CertChainFiles, fileBundle.RootCertFile); err != nil {
```
[the remark this answers is not in the document]
Comment (nmnellis) [reply security/pkg/pki/ca/ca.go:251]: i think this is good, the util package doesnt use the file bundle

**Pull request merged**